### ISSUE TITLE
Fillable forms with default/preselected buttons

### ISSUE THREAD

**Issue by kberry** [ho-tex/hyperref#7]
(reposted from https://puszcza.gnu.org.ua/bugs/index.php?202, author Thorsten <zittho>)

Hi. I am trying to generate forms with preselected buttons using the "default" parameter. This works for the first example/button but not for the second example/button.
```

\documentclass{article}
\usepackage{hyperref}

\begin{document}
\begin{Form}
\ChoiceMenu[default=y,radio,name=atesta]{Question:}{yes=y,no=n}
\end{Form}

\begin{Form}
\ChoiceMenu[default=n,radio,name=btestb]{Question:}{yes=y,no=n}
\end{Form}
\end{document}
```

How can I fix this?

Thx, Thorsten

Program versions are:

Hyperref 2012/11/06 v6.83m

$ pdflatex -v
pdfTeX 3.1415926-2.5-1.40.14 (TeX Live 2013/Debian)
kpathsea version 6.1.1
Copyright 2013 Peter Breitenlohner (eTeX)/Han The Thanh (pdfTeX).
There is NO warranty. Redistribution of this software is
covered by the terms of both the pdfTeX copyright and
the Lesser GNU General Public License.
For more information about these matters, see the file
named COPYING and the pdfTeX source.
Primary author of pdfTeX: Peter Breitenlohner (eTeX)/Han The Thanh (pdfTeX).
Compiled with libpng 1.2.49; using libpng 1.2.49
Compiled with zlib 1.2.8; using zlib 1.2.8
Compiled with poppler version 0.24.3

-- (comment #1)

I played around with pdftk and the uncompressed pdf. It seems that for every radio button both "/V" entries must have the default value.

The following "dirty" patch works for me.

--- hpdftex.def.bak 2014-01-14 10:31:34.050531142 +0100
+++ hpdftex.def 2014-01-14 11:03:08.438525051 +0100
@@ -1607,8 +1607,8 @@
/V/\Fld@default
/DV/\Fld@default
\else
- /V/Off%
- /DV/Off%
+ /V/\Fld@default
+ /DV/\Fld@default
\fi
\Fld@additionalactions
}

**Comment by u-fischer** [comment]
See my second comment in https://github.com/ho-tex/hyperref/issues/6

**Comment by davidcarlisle** [comment]
fixed by code for issue #6 I believe.
hyperref v6.83p uploaded to CTAN
